src/main/kotlin/game/racing/domain/CarFactory.kt

```diff
@@ -0,0 +1,11 @@
+package game.racing.domain
+
+object CarFactory {
+    fun generateCarByNumberOfCar(numberOfCar: Int): List<Car> {
+        val cars = mutableListOf<Car>()
+        repeat(numberOfCar) {
+            cars.add(Car())
+        }
+        return cars
```

Comment:
아래와 같이 동일한 코드를 조금 더 간단하게 변경해볼 수 있을 것 같아요. 😉
```suggestion
        return List(numberOfCar) { Car()}
```